src/rpc/rawtransaction.cpp

```diff
@@ -211,7 +211,7 @@ UniValue gettxoutproof(const UniValue& params, bool fHelp)
             "\nNOTE: By default this function only works sometimes. This is when there is an\n"
             "unspent output in the utxo for this transaction. To make it always work,\n"
             "you need to maintain a transaction index, using the -txindex command line option or\n"
-            "specify the block in which the transaction is included in manually (by blockhash).\n"
+            "specify the block in which the transaction is included manually (by blockhash).\n"
```

Comment:
Maybe. My point is that the word 'in' is there twice and shouldn't be.

Either

> "the block which the tx is included **in**"

or

> "the block **in** which the tx is included"

are OK but

> "the block **in** which the tx is included **in**"

isn't.